scripts/barclamp_install.rb

```diff
@@ -54,10 +54,13 @@ def usage()
 
 usage if ARGV.length < 1
 
-tmpdir = "/tmp/bc_install-#{Process.pid}-#{Kernel.rand(65535)}"
+tmpdir = "/tmp/component_install-#{Process.pid}-#{Kernel.rand(65535)}"
 debug "tarball tmpdir: #{tmpdir}"
 Dir.mkdir(tmpdir)
-candidates = Array.new
+
+log_path = File.join '/var', 'log', 'crowbar'
+FileUtils.mkdir log_path unless File.directory? log_path
+log = File.join log_path, "component_install_#{Time.now.strftime("%Y%m%dT%H%M")}.log"
```

Comment:
Hrm, I really dislike that. Just append to a log file without a date in the filename, otherwise we'll end up with thousands of such files.

Also we need to make sure to have the correct logrotate rule.